terraform/azure/mds-existing-vnet/README.md

```diff
@@ -0,0 +1,177 @@
+# Check Point CloudGuard Network Security MDS Terraform deployment for Azure
+
+This Terraform module deploys Check Point CloudGuard IaaS Management solution into a new Vnet in Azure.
+As part of the deployment the following resources are created:
+- Resource group
+- Network security group
+- Virtual Machine
+- System assigned identity
+
+This solution uses the following modules:
+- /terraform/azure/modules/common - used for creating a resource group and defining common variables.
+- /terraform/azure/modules/network-security-group - used for creating new network security groups and rules.
+
+
+## Configurations
+- Install and configure Terraform to provision Azure resources: [Configure Terraform for Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/terraform-install-configure)
+- In order to use ssh connection to VMs, it is **required** to add a public key to the /terraform/azure/mds-existing-vnet/azure_public_key file.
+
+## Usage
+- Choose the preferred login method to Azure in order to deploy the solution:
+    <br>1. Using Service Principal:
+    - Create a [Service Principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal) (or use the existing one) 
+    - Grant the Service Principal at least "**Owner**" permissions to the Azure subscription<br>
```

Comment:
For MDS we don't need owner permission.
Please update to the most restrictive role needed.